appveyor.yml

```diff
@@ -1,10 +1,6 @@
 image: Visual Studio 2017
 clone_folder: 'c:\gopath\src\github.com\%APPVEYOR_REPO_NAME%'
 
-branches:
-  only:
-    - /^v\d+\.\d+\.\d+$/
```

Comment:
Wasn't this necessary so AppVeyor doesn't build every single branch?

Reply:
Yes, but I have to find a different way to avoid that, the build for the tag 'v0.22.0' didn't trigger because of that. It really means only branches, nothing else.